### PR TITLE
Add Font Awesome Pro to the site

### PR DESCRIPTION
**Pull request by levadadenys**
## Links
[Font Awesome Docs](https://fontawesome.com/docs/web/setup/use-kit)

Jira ticket: [SL-339](https://codedotorg.atlassian.net/browse/SL-339)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/hourofcode2022.haml [hunk 1]
```diff
@@ -6,9 +6,6 @@ theme: responsive
 %link{href: "/css/generated/hoc2022-landing-page.css", rel: "stylesheet"}
 %link{href: "/css/hoc2022-nasa-skinny-banner.css", rel: "stylesheet"}
 
--# Import FontAwesome Pro
-%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
-
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/hourofcode2022.haml:11]: @levadadenys thanks for cleaning this up! 🙌

**Review by kelbyhawn — APPROVED** [review]
The pegasus changes look great to me 🎉 

Before you merge, it would be good to have someone on the Student Learning team take a look at the dashboard/app changes since they'll be more familiar with how that works.

File: dashboard/app/views/layouts/_font_awesome_pro.html.haml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+-# Import FONT AWESOME PRO for all dashboard pages
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous", async: true}
```
Comment (breville) [inline dashboard/app/views/layouts/_font_awesome_pro.html.haml:2]: This takes a dependency on a third-party domain, which we should avoid doing, especially for something so essential.  We should self-host all related content for this change.

Comment (levadadenys) [reply dashboard/app/views/layouts/_font_awesome_pro.html.haml:2]: Good point, thank you! I've found a way to import font awesome pro kit via ruby, just let me find the token for our kit and I'll update the PR

**Comment by levadadenys** [comment]
I've created a new PR with this task(this one is based on my own fork of repo, and the new one is based on code.org original repo), please take a look [https://github.com/code-dot-org/code-dot-org/pull/49406](url).

**Pull request closed without merging**